.typedoc/custom-theme.mjs

```
@@ -46,6 +46,27 @@ class ClerkMarkdownThemeContext extends MarkdownThemeContext {
+      comment: (model, options) => {
+        if (
+          model.hasModifier('@experimental') &&
+          [ReflectionKind.Class, ReflectionKind.Interface].includes(this.page?.model?.kind)
+        ) {
+          model.removeModifier('@experimental');
+          model.removeTags('@example');
+          model.removeTags('@see');
+
+          const res = superPartials.comment(model, options);
+
+          const [line, ...linesToInclude] = res.replace(/^\n\n/, '').split('\n');
```

ESLint

[error] 'line' is assigned a value but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)